docs/span-compression.asciidoc

```diff
@@ -61,5 +61,28 @@ Spans with longer duration are not compressed. Please refer to the agent documen
 
 Support for span compression is available in these agents:
 
- * Go: {apm-go-ref}/configuration.html#config-span-compression-exact-match-duration[`ELASTIC_APM_SPAN_COMPRESSION_EXACT_MATCH_MAX_DURATION`], {apm-go-ref}/configuration.html#config-span-compression-same-kind-duration[`ELASTIC_APM_SPAN_COMPRESSION_SAME_KIND_MAX_DURATION`]
- * Python: {apm-py-ref}/configuration.html#config-span-compression-exact-match-max_duration[`span_compression_exact_match_max_duration`], {apm-py-ref}/configuration.html#config-span-compression-same-kind-max-duration[`span_compression_same_kind_max_duration`]
\ No newline at end of file
+[options="header"]
+|====
+| Agent | Same-kind config | Exact-match config
+| **Go agent**
+| {apm-go-ref-v}/configuration.html#config-span-compression-same-kind-duration[`ELASTIC_APM_SPAN_COMPRESSION_SAME_KIND_MAX_DURATION`]
+| {apm-go-ref-v}/configuration.html#config-span-compression-exact-match-duration[`ELASTIC_APM_SPAN_COMPRESSION_EXACT_MATCH_MAX_DURATION`]
+| **Java agent**
+| {apm-java-ref-v}/config-huge-traces.html#config-span-compression-same-kind-max-duration[`span_compression_same_kind_max_duration`]
+| {apm-java-ref-v}/config-huge-traces.html#config-span-compression-exact-match-max-duration[`span_compression_exact_match_max_duration`]
+// | **.NET agent**
+// | {apm-dotnet-ref-v}/config-span-compression-same-kind-max-duration.html[`SpanCompressionSameKindMaxDuration`]
+// | {apm-dotnet-ref-v}/config-span-compression-exact-match-max-duration.html[`SpanCompressionExactMatchMaxDuration`]
+| **Node.js agent**
+| {apm-node-ref-v}/configuration.html#span-compression-same-kind-max-duration[`spanCompressionSameKindMaxDuration`]
+| {apm-node-ref-v}/configuration.html#span-compression-exact-match-max-duration[`spanCompressionExactMatchMaxDuration`]
+// | **PHP agent**
+// | {apm-php-ref-v}[``]
+// | {apm-php-ref-v}[``]
```

Comment:
@SergeyKleyman are there docs for this feature? I wasn't able to find a PR that matches the issue (https://github.com/elastic/apm-agent-php/issues/425).